### PR TITLE
Fix getting digest from oci images build with buildkit

### PR DESCRIPTION
**Pull request by gsanchezgavier**
Reroller is failing when trying to get digest from Images build with buildkit 

The pr fixes that, bumps deps and publish image in github packages

```
time="2023-03-22T08:15:19Z" level=error msg="Could not fetch latest digest for docker.io/newrelic/infrastructure-bundle:latest: getting manifest list from docker: docker API did not return a content digest for this image"

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -3,23 +3,28 @@ on:
...
       - run: |
           DOCKER_IMAGE_TAG=$(echo "${{ github.event.release.tag_name }}" | sed 's/^v//')
           echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
-      - uses: docker/login-action@v1
+      - uses: docker/login-action@v2
         with:
-          username: roobre
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
```
Comment (kilokang) [inline .github/workflows/release.yml]:
```suggestion
          username: ${{ github.repository_owner }}
```

File: go.mod [hunk 2]
```diff
@@ -1,12 +1,64 @@
 module roob.re/reroller
```
Comment (kilokang) [inline go.mod:1]: NIT
```suggestion
module github.com/newrelic-forks/reroller
```

File: go.mod [hunk 3]
```diff
@@ -1,12 +1,64 @@
...
 
-go 1.16
+go 1.20
```
Comment (kilokang) [inline go.mod]: As we where having issues with Alpine in other integrations, I would keep this to 1.19.

**Review by kilokang — COMMENTED** [review]
Way cleaner.

I left a comment on the pipeline that you used "actor" instead of "repository owner".

File: go.mod (hunk 3 above)
Comment (gsanchezgavier) [reply go.mod]: oh forgot that , thanks

File: go.mod (hunk 2 above)
Comment (gsanchezgavier) [reply go.mod:1]: i would have to change the imports to, not sure if this is needed.

Comment (kilokang) [reply go.mod:1]: I think it is not. that is why it is an NIT.

**Review by kilokang — APPROVED** [review]
I would wait for the creator of this to give his two cents before leaving :)

From my side, LGTM.

**Pull request merged**